src/main/java/com/denizenscript/denizencore/tags/core/UtilTagBase.java

```diff
@@ -1018,11 +1031,10 @@ public static void adjustSystem(Mechanism mechanism) {
         // Note that this redirects *all console output* not just Denizen output.
         // Note: don't enable /denizen debug -e while this is active.
         // Requires config file setting "Debug.Allow console redirection"!
-        // For example: - adjust system redirect_logging:true
```

Comment:
Why was this just deleted? Should be replaced with an `@example`

Reply:
Removed it since the auto-generated example is the same, let me know if I should re-add the manual one

Reply:
auto-generated examples are randomized and considered unverified even if they happen to be correct.
We're aiming for having manual examples on all meta entries eventually

Reply:
Re-added it